tar/util.c

```diff
@@ -140,6 +140,7 @@ safe_fprintf(FILE *f, const char *fmt, ...)
 		} else {
 			/* Leave fmtbuff pointing to the truncated
 			 * string in fmtbuff_stack. */
+			fmtbuff = fmtbuff_stack;
```

Comment:
This is technically redundant (since `fmtbuff` has not been changed since it was set in line 111 above.

But I agree it's a little clearer to set it again here.